react/features/toolbox/components/native/AudioOnlyButton.js

```diff
@@ -41,9 +50,19 @@ class AudioOnlyButton extends AbstractButton<Props, *> {
      * @returns {void}
      */
     _handleClick() {
-        this.props.dispatch(toggleAudioOnly());
+        const { _audioOnly, _startCarMode, dispatch } = this.props;
+
+        if (!_audioOnly && _startCarMode) {
+            dispatch(setAudioOnly(true));
+            navigate(screen.conference.carmode);
+        } else if (_audioOnly && _startCarMode) {
```

Comment:
I think you can safely drop this branch and just have the else.

Reply:
Tested it and works. Will update.